Update integrations instructions (#1259)
neondatabase/website: content/docs/manage/integrations.md

When visiting the **Integrations** page in your console, you'll notice different categories:

> - **Manage**: These are integrations added to your project.

Reviewer: ```suggestion
- **Manage**: A list of integrations that have already been added to your project.
```

> - **Add**: These are integrations available for you to add and use immediately.

Reviewer: ```suggestion
- **Add**: Integrations that are available for you to add and use immediately.
```

> - **Request**: These are the integrations that will be available in the near future.

Reviewer: ```suggestion
- **Request**: Upcoming integrations that you can express interest in to help prioritize their release.
```
